server/db/add_hubs.sh

```diff
@@ -17,7 +17,6 @@ elif [[ ! -n "$2" ]]; then
     echo "The hubs CSV file"
     echo "MUST have the header 'system_id, system_name, heartbeat_alert_recipients'"
     echo "MUST use Unix line endings (LF)"
-    echo "MUST end with a newline, or else the last hub will be silently ignored"
```

Comment:
Maybe worth adding a message saying that it `MUST not end with a newline` ? 

On the other hand, it find this inconsistent because I always have a newline at the end of my CSV that adds new buttons. I've never tried it without the newline so am not sure whether it would ignore the last Buttons or not. Is there something different about the loop in `add_buttons_to_installation.sh`?

Reply:
I think the difference is that `add_buttons_to_installation.sh` is checking the variable that is in the third position in the CSV format while `add_hubs.sh` is checking the variable in the first position.